src/lib/selectors/youtube/select.py

```diff
@@ -83,15 +83,14 @@ def str2bool(v):
 def youtube_search_all_pages(args, logs):
     csv_obj = []
     print_log(
-        "Search terms: %s\n Start: %s\n End: %s"
-        % (args["q"], args["after"], args["before"]),
+        f"Search terms: {args["q"]}\n Start: {args["after"]}\n End: {args["before"]}",
```

Comment:
I've tried running this, and it introduces errors due to the nested double quotes across this file. (This is partly our fault, as we don't yet have tests and CI setup-- i.e. #12, #13, #14, plus the configuration documentation needs some work.) Would you be able to change this and the other instances in this file?

For example, this should become:
```python
f"Search terms: {args['q']}\n Start: {args['after']}\n End: {args['before']}",
```

Reply:
You are absolutely right, this should be changed.

Reply:
Note that there are a couple of other instances in this file, youtube/select.py, where this same nested double quote needs to be fixed. Sorry if my comment before wasn't clear!

Reply:
My bad. Should have checked it at first.

Reply:
Applied flake8 and pylint this time. Hope it will be all good. 😅